doc/pages/Distribution.md

```diff
@@ -0,0 +1,54 @@
+| Debian Linux (9,stretch)      | 1.2.2       | Mehdi Dogguy [@mehdid](https://github.com/mehdid), Nicolas Braud-Santoni [@nbraud](https://www.github.com/nbraud) | [Package search](https://packages.debian.org/search?keywords=opam&searchon=names&suite=all&section=all)
+| Debian Linux (10,buster)      | 2.0.3       | Mehdi Dogguy [@mehdid](https://github.com/mehdid), Nicolas Braud-Santoni [@nbraud](https://www.github.com/nbraud) | [Package search](https://packages.debian.org/search?keywords=opam&searchon=names&suite=all&section=all)
+| Debian Linux (unstable, sid)  | 2.0.3       | Mehdi Dogguy [@mehdid](https://github.com/mehdid), Nicolas Braud-Santoni [@nbraud](https://www.github.com/nbraud) | [Package search](https://packages.debian.org/search?keywords=opam&searchon=names&suite=all&section=all)
+| Ubuntu Linux (14.04,trusty)   | 1.1.1       | -                                                                                                                 | [Package search](http://packages.ubuntu.com/search?keywords=opam&searchon=names&suite=all&section=all) - bwrap unavailable
+| Ubuntu Linux (16.04,xenial)   | 1.2.2       | -                                                                                                                 | [Package search](http://packages.ubuntu.com/search?keywords=opam&searchon=names&suite=all&section=all) - bwrap unavailable
+| Ubuntu Linux (17.04,artful)   | 1.2.2       | -                                                                                                                 | [Package search](http://packages.ubuntu.com/search?keywords=opam&searchon=names&suite=all&section=all)
+| Ubuntu Linux (18.04,bionic)   | 1.2.2       | -                                                                                                                 | [Package search](http://packages.ubuntu.com/search?keywords=opam&searchon=names&suite=all&section=all)
+| Ubuntu Linux (19.04,cosmic)   | 1.2.2       | -                                                                                                                 | [Package search](http://packages.ubuntu.com/search?keywords=opam&searchon=names&suite=all&section=all)
+| Ubuntu Linux (19.04,disco)    | 2.0.3       | -                                                                                                                 | [Package search](http://packages.ubuntu.com/search?keywords=opam&searchon=names&suite=all&section=all)
+| Ubuntu Linux PPA              | 2.0.4       | Anil Madhavapeddy [@avsm](https://www.github.com/avsm)                                                            | [Anil's official OPAM PPA](https://launchpad.net/~avsm) and [ppa post](https://discuss.ocaml.org/t/opam-2-0-experimental-ppas/2446)
+| Homebrew (MacOS X)            | 2.0.0       | Anil Madhavapeddy [@avsm](https://www.github.com/avsm)                                                            |
```

Comment:
```suggestion
| Homebrew (MacOS X)            | 2.0.4       |                                                            |
```

updated version, and homebrew has no official maintainer system I think, so removing my name